exporter.go

```diff
@@ -274,7 +274,7 @@ func (b *Exporter) Listen(e <-chan Events) {
 				// We don't accept negative values for counters. Incrementing the counter with a negative number
 				// will cause the exporter to panic. Instead we will warn and continue to the next event.
 				if event.Value() < 0.0 {
-					log.Errorf("Counter %q is: '%f' (counter must be non-negative value)", metricName, event.Value())
+					log.Debugf("Counter %q is: '%f' (counter must be non-negative value)", metricName, event.Value())
```

Comment:
This one doesn't seem to have a metric associated with it.  But, I don't think it's that important.